stone/backends/python_helpers.py

```diff
@@ -77,6 +77,27 @@ def fmt_var(name, check_reserved=False):
     s = fmt_underscores(name)
     return _rename_if_reserved(s) if check_reserved else s
 
+def fmt_version(version, prefix='_v'):
+    if version == 1:
+        return ''
+    else:
+        return '{}{}'.format(prefix, version)
+
+def check_route_name_conflict(namespace):
+    """
+    Check name conflicts among generated route definitions. Raise a runtime exception when a
+    conflict is encountered.
+    """
+
+    route_by_name = {}
+    for route in namespace.routes:
+        route_name = '{}{}'.format(route.name, fmt_version(route.version))
```

Comment:
Can we move `'{}{}'.format(route.name, fmt_version(route.version))` to some helper function so that we don't have to do so repeatedly?

Reply:
I don't see much benefits of doing this. The only logic would be to take two strings and concatenate them together. It's too simple to worth a separate function.